packages/components/src/docs/Migration/ui-tag.mdx

```diff
@@ -0,0 +1,43 @@
+# `ui-tag`
+
+The new `sd-tag` is designed to replace the `ui-tag` and provide enhanced functionality. To understand all the available attributes, please refer to the `sd-tag` documentation.
+
+## ✨ New attributes: `href`, `target` and `download`
+
+When the new `href` attribute is set the `sd-tag` component renders as a link (`<a>`) and not (as per default) as a button (`<button>`). The additional `target` attribute controls where the browser should open the link, the `download` attribute may be used to tell the browser to download the linked file with the specified filename. If the `href` attribute is omitted, the `target` and `download` attribute are ignored.
+
+**Hint:** When the `href` attribute is set i.e. the `sd-tag` is rendered as a link (`<a>`), the `disabled` attribute is ignored, as links may not be disabled. To disable the `sd-tag` the `href` attribute has to be removed as well.
+
+## ✨ New attribute: `removable`
+
+The new `removable` attribute controls whether or not the tag should display a small close icon (`x`) next to the label of the tag to indicate that the filtered attribute may be removed.
+
+## ✨ New attribute: `size`
+
+The `sd-tag` component offers two different sizes:
+
+- `lg` is the default size
+- `sm` size is an alternative size for use in very tight spaces
+
+## ❌ Removed CSS Variables
+
+Component-specific CSS variables have been discontinued. Now, you can override colors and padding using the part selector.
+
+```css
+sd-tag::part(base) {
+  background: pink;
+  color: black;
+}
+```
+
+## ❌ Removed default click handler
+
+The `sd-tag` component does not provide a default handler/behavior for click events. Therefore, the `value` attribute has been removed as well.
+
+## ❌ Removed attribute: `labelId`
+
+The `sd-tag` no longer has a label part and therefore does not provide a `labelId` attribute.
+
+## 🔀 Updated attribute: `checked` to `selected`
```

Comment:
I believe in "checked" the "x" was always shown. Please add some information, that this now has to be added additionally with the `removable` attribute.

Reply:
```suggestion
## 🔀 Updated attribute: `checked` to `selected` and `removable`
```

You could do it like this and combine the two paragraphs as this shows how to upgrade.